index.html

```diff
@@ -0,0 +1,147 @@
+            <div class="main_program_cards">
+            <div class="main_program_card">
+                <i class="fas fa-handshake fa-2x"></i>
+                <strong>Network</strong>
+                <p>
+                    Network and interact with top industry individuals with many years of experience.
+            </p>
+            </div>
+            <div class="main_program_card">
+                <i class="fas fa-handshake fa-2x"></i>
+                <strong>Network</strong>
+                <p>
+                    Network and interact with top industry individuals with many years of experience.
+            </p>
+            </div>
+            <div class="main_program_card">
+                <i class="fas fa-handshake fa-2x"></i>
+                <strong>Network</strong>
+                <p>
+                    Network and interact with top industry individuals with many years of experience.
+            </p>
+            </div>
+            <div class="main_program_card">
+                <i class="fas fa-handshake fa-2x"></i>
+                <strong>Network</strong>
+                <p>
+                    Network and interact with top industry individuals with many years of experience.
+            </p>
+            </div>
+            <div class="main_program_card">
+                <i class="fas fa-handshake fa-2x"></i>
+                <strong>Network</strong>
+                <p>
+                    Network and interact with top industry individuals with many years of experience.
+            </p>
+            </div>
+        </div>
+        <a class="desktop_see_program" href="#">See the hall Program</a>    
+        <div class="main_program_button">Join the Course</div>
+        </section>
+        <section class="speakers_section" id="speakers">
+            <h3 class="speakers_section_title">Featured Speakers</h3>
+            <hr/>
+            <div  class="speaker_card_list">
+            
+           </div>
+           <div  class="speaker_card_list1">
+            
+        </div>  
+            <div class="speakers_more">
+                  MORE
+                 <i class="fas fa-angle-down" ></i>
+            </div>
+        </section>
+
+        <!--desktop only on desktop-->
+        <section class="home_partners" id="home_partners">
+            <h3 class="partners_title">Partners</h3>
+            <hr/>
+            <div class="partners_icons"> 
+                
+                <div class="partners_icon"><i class="fab fa-facebook-square"></i>acebook</div>
+                <div class="partners_icon"><i class="fab fa-google"></i>oogle</div>
+                <div class="partners_icon"><i class="fab fa-uber"></i>Uber</div>
+                <div class="partners_icon"><i class="fab fa-amazon"></i>mazone</div>
+                <div class="partners_icon">Oracle</div>
+                
+            </div>
+         </section>
+         <footer id="home_footer">    
```

Comment:
- [ ] **Required** In the Mobile version, there should be a footer on the `home page` same as `about page` with white background and logo on the left side.

Reply:
thanks for reviewing and for your time ,i checked the given template so the mobile home page doesn't have a footer